### PR TITLE
Strip out all special characters from csv headers

### PR DESCRIPTION
**Pull request by orangewolf**
These cause nothing but pain.

### CODE REVIEW

File: app/models/bulkrax/csv_entry.rb [hunk 1]
```diff
@@ -16,11 +16,12 @@ def self.fields_from_data(data)
     class_attribute(:csv_read_data_options, default: {})
 
     # there's a risk that this reads the whole file into memory and could cause a memory leak
+    # we strip any special characters out of the headers. looking at you Excel
     def self.read_data(path)
       raise StandardError, 'CSV path empty' if path.blank?
       options = {
         headers: true,
-        header_converters: ->(h) { h.to_s.strip.to_sym },
+        header_converters: ->(h) { h.to_s.gsub(/[^\w\d -]+/, '').strip.to_sym },
```
Comment (jeremyf) [inline app/models/bulkrax/csv_entry.rb:24]: Do we want underscore as well?

Comment (orangewolf) [reply app/models/bulkrax/csv_entry.rb:24]: @jeremyf \w includes underscore.

**Comment by orangewolf** [comment]
closing this one for #918 instead as it is a cleaner PR

File: app/models/bulkrax/csv_entry.rb (hunk 1 above)
Comment (jeremyf) [reply app/models/bulkrax/csv_entry.rb:24]: Today I (re-)learned!

**Pull request closed without merging**